bin/amiupdater

```diff
@@ -1 +1,38 @@
-# Placeholder for future development.
+#!/usr/bin/env python
```

Comment:
can as much logic as possible be removed from this and added to the AMIUpdate class as static methods ? Will make it easier to test those components and leave a smaller coverage gap if we don't have tests for this file. Will also make integration into v9 easier.